frontend/catalyst/jit.py

```diff
@@ -384,6 +384,7 @@ def compute_jvp(self, primals, tangents):
         # Optimization: Do not compute Jacobians for arguments which do not participate in
         #               differentiation.
         argnums = []
+        tangents, _ = tree_flatten(tangents)
```

Comment:
Added for the jax version in the test to run

Reply:
Oh, didn't we want to swap this implementation in Python for the `JVP` op? 😅

Reply:
Yes I think we wanted but I don't recall why we stopped, but there was a reason I think.